### ISSUE TITLE
别让Lombok使你的Java代码处于“亚健康”状态 · 树下魅狐

### ISSUE THREAD

**Issue by ramostear** [ramostear/ramostear-comment#134]
https://www.ramostear.com/blog/2020/04/28/uk1860p8.html 

 如果您正在阅读此文，想必您对Project Lombok已经有了一段时间的了解。您是否正准备拥抱Lombok?还是正准备将如此酷炫的项目推荐给你的团队？如果您准备那么做，不妨听听我在使用Lombok一年后的一些感受。

**Comment by light0x00** [comment]
确实, 但是单纯的使用 @getter、@setter 还是可以的,  @data做的事太多了,hash/equals这种随便重写肯定是不行的.

**Comment by ramostear** [comment]
我遇到的问题主要是data注解带来的，例如枚举类发自我的华为手机-------- 原始邮件 --------发件人： light <notifications@github.com>日期： 2020年4月28日周二 傍晚6:43收件人： ramostear/ramostear-comment <ramostear-comment@noreply.github.com>抄送： ramostear <ramostear@163.com>, Author <author@noreply.github.com>主    题： Re: [ramostear/ramostear-comment] 别让Lombok使你的Java代码处于“亚健康”状态 · 树下魅狐 (#134)

确实, 但是单纯的使用 @getter、@Setter 还是可以的,  @DaTa做的事太多了,hash/equals这种随便重写肯定是不行的.

**Comment by zhengchalei** [comment]
3. 可读性差

构造器 建议自己重写, 他也不会覆盖你写的方法, 把 Lombok 当做一个比较纯粹的东西就不会有那么多烦恼了
